src/demo/adagrams.js

```diff
@@ -3,13 +3,15 @@ import {
   usesAvailableLetters,
   scoreWord,
   highestScoreFrom,
+  createLetterPool,
```

Comment:
We could remove `createLetterPool` from the import lists if it isn't being directly used by this file.